concent_api/verifier/utils.py

```diff
@@ -52,30 +52,41 @@ def prepare_storage_request_headers(file_transfer_token: message.FileTransferTok
 
 
 def store_file_from_response_in_chunks(response: requests.Response, file_path: str):
-    with open(file_path, 'x') as f:
+    with open(file_path, 'wb') as f:
```

Comment:
Why not 'x' ? It is supposed to fail when file already exists.

Reply:
actually should be `'xb'`, as it is a binary file, will change it ;)

Reply:
done